RecoTracker/LSTCore/interface/ObjectRangesSoA.h

```diff
@@ -0,0 +1,51 @@
+#ifndef RecoTracker_LSTCore_interface_ObjectRangesSoA_h
+#define RecoTracker_LSTCore_interface_ObjectRangesSoA_h
+
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
+#include "DataFormats/Portable/interface/PortableCollection.h"
+
+namespace lst {
+
+  GENERATE_SOA_LAYOUT(ObjectRangesSoALayout,
+                      SOA_COLUMN(ArrayIx2, hitRanges),
+                      SOA_COLUMN(int, hitRangesLower),
+                      SOA_COLUMN(int, hitRangesUpper),
+                      SOA_COLUMN(int8_t, hitRangesnLower),
+                      SOA_COLUMN(int8_t, hitRangesnUpper),
+                      SOA_COLUMN(ArrayIx2, mdRanges),
+                      SOA_COLUMN(ArrayIx2, segmentRanges),
+                      SOA_COLUMN(ArrayIx2, trackletRanges),
+                      SOA_COLUMN(ArrayIx2, tripletRanges),
+                      SOA_COLUMN(ArrayIx2, quintupletRanges),
+                      SOA_COLUMN(ArrayIx2, trackCandidateRanges))
+
+  // triplets and quintuplets end up with an ununsed pixel entry at the end
+  GENERATE_SOA_LAYOUT(ObjectRangesOccupancySoALayout,
```

Comment:
```suggestion
  GENERATE_SOA_LAYOUT(ObjectOccupancySoALayout,
```
I tend to think that `Ranges` is not needed if `Occupancy` is used.

Reply:
Yeah, that sounds better